Framework/Beamline/src/DetectorInfo.cpp

```diff
@@ -9,6 +9,8 @@
 #include "MantidKernel/make_cow.h"
 
 #include <algorithm>
+#include <exception>
+#include <iostream>
```

Comment:
I believe `iostream` features are not used in your changes, could you remove it?

Reply:
Right, this include was there for debugging purposes. I removed it.